src/main/java/seedu/resireg/model/alias/AliasUtils.java

```diff
@@ -0,0 +1,20 @@
+package seedu.resireg.model.alias;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Utility functions for classes in the model.alias package.
+ */
+public class AliasUtils {
+    /**
+     * Converts a list of {@code CommandWordAlias} to a map of alias to command word.
+     */
+    public static Map<String, String> makeAliasToCommandWordMap(List<CommandWordAlias> aliasList) {
+        return aliasList.stream().collect(Collectors.toMap(
+            commandWordAlias -> commandWordAlias.alias.alias,
```

Comment:
Can we change this to `commandWordAlias.alias.toString()` to follow the Law of Demeter?

Reply:
Alright changed it, thanks for the catch